src/sparsezoo/model/model.py

```diff
@@ -156,7 +157,14 @@ def __init__(self, source: str, download_path: Optional[str] = None):
             stub_params=self.stub_params,
         )
 
-        self.onnx_model: File = self._file_from_files(files, display_name="model.onnx")
+        self._onnx_gz: Directory = self._directory_from_files(
+            files, directory_class=OnnxGz, display_name="model.onnx.tar.gz"
+        )
+        self.onnx_model: File = (
+            self._file_from_files(files, display_name="model.onnx.")
```

Comment:
should this be "model.onnx" instead of "model.onnx." ?

Reply:
good catch